dotCMS/src/integration-test/java/com/dotcms/contenttype/business/FieldAPITest.java

```diff
@@ -1357,6 +1362,129 @@ public void test_SaveField_GivenGraphQLCompatibleVariable_ShouldSave(
         }
     }
 
+    /**
+     * <b>Method to test:</b> {@link FieldAPIImpl#save(Field, User)}
+     * <p>
+     * <b>Given scenario:</b> A new {@link Field} is saved and marked as `indexed=false`
+     * <p>
+     * <b>Expected result:</b> The field should be saved without ES mapping
+     */
+    @Test
+    public void test_SaveNewNoIndexedField_ShouldNotAddESMapping()
+            throws DotSecurityException, DotDataException, IOException {
+        final long time = System.currentTimeMillis();
+        final ContentType type = new ContentTypeDataGen().nextPersisted();
+        try {
+            Field field = FieldBuilder.builder(DateField.class)
+                    .name("field" + time)
+                    .variable("field" + time)
+                    .contentTypeId(type.id())
+                    .indexed(false)
+                    .build();
+            field = fieldAPI.save(field, user);
+
+            final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+            final ESMappingAPIImpl mappingAPI = new ESMappingAPIImpl();
+
+            //verify mapping on working index
+            Map<String, Object> mapping = mappingAPI
+                    .getFieldMappingAsMap(indiciesInfo.getWorking(),
+                            (type.variable() + StringPool.PERIOD + field.variable()));
+            assertFalse(UtilMethods.isSet(mapping));
+
+            //verify mapping on live index
+            mapping = mappingAPI
+                    .getFieldMappingAsMap(indiciesInfo.getLive(),
+                            (type.variable() + StringPool.PERIOD + field.variable()));
+            assertFalse(UtilMethods.isSet(mapping));
+        }finally{
+            ContentTypeDataGen.remove(type);
+        }
+    }
+
+    /**
+     * <b>Method to test:</b> {@link FieldAPIImpl#save(Field, User)}
+     * <p>
+     * <b>Given scenario:</b> A new {@link DateField} is saved and marked as `indexed=true`
+     * <p>
+     * <b>Expected result:</b> The field should be saved and mapped in ES with `type=date`
```

Comment:
maybe we can test other data types and their expected mapping

Reply:
those cases have been already tested